### ISSUE TITLE
Snapchat fails to send Videos while send gallery to snaps in turned on

### ISSUE THREAD

**Issue by RevealedSoulEven** [rodit/SnapMod#256]
Snapchat can't send any video while the feature is on. It says "Failed to send snap". And not only the camera roll videos but also the videos recorded from the Snapchat camera itself.

Attaching the logs

[modules_2022-09-24T06_18_24.286.log](https://github.com/rodit/SnapMod/files/9637613/modules_2022-09-24T06_18_24.286.log)

**Comment by RevealedSoulEven** [comment]
Adding to this.... The error comes even I turned off all the features related to snaps like save snaps/save stories/disable snap splitting/send gallery as snaps

Then also the same problem is coming.

I then turned off snapmod from lsposed modules to check whether it's a Snapchat bug. But the recorded videos are sent as snaps that are recorded from Snapchat in-app camera. But when SnapMod is turned on. It fails to send(in-app camera and camera roll both)

**Comment by melbarouzi** [comment]
U can see which  Feature are working or not in the wiki [link](https://github.com/rodit/SnapMod/wiki/Features) gived in last update discussion 

And sending snaps from gallery is indicated that it will not work with the latest update ... i hope @rodit  will fix Those bugs asap

**Comment by RevealedSoulEven** [comment]
But the thing is even without using any feature related to snaps, then also normal videos aren't being sent

**Comment by Lucky-069** [comment]
Perhaps I think a switch is needed in snapmod to turn off snapmod and sort of unlink it from the official snap and then turn it back on when needed. Since sending video works on normal snap.

**Comment by RevealedSoulEven** [comment]
No bro. It's a bug which I noticed is occurring due to snapmod from snapchat 11.93.xx.xx update because I used to build the mappings myself so I know it.
Everything is working fine if I use snapchat 11.92.0.33 with mappings(self made)

**Comment by Lucky-069** [comment]
Yea but I'm suggesting something that can be used later too

**Comment by tycoon420** [comment]
> No bro. It's a bug which I noticed is occurring due to snapmod from snapchat 11.93.xx.xx update because I used to build the mappings myself so I know it. Everything is working fine if I use snapchat 11.92.0.33 with mappings(self made)

yeah i have the exactly same bug. In the new Snapchat 11.96.   . Privacy policy is changed with some new feautures. So the feauture of sending gallery video as snap isn't working. Snap failed to be sent is always shown on mine too. So i changed back my Snapchat to older version and used old mapping of 84615 which for Snapchat 11.82.0.32. Sending gallery video is working fine on the older build. Here is the mapping of 11.82.0.32 attached below. Unzip and Just copy it to/data/data/xyz.rodit.snapmod/files/. For time being i don't have any issue with older version. It works perfectly fine.
[84615.zip](https://github.com/rodit/SnapMod/files/9638416/84615.zip)

**Comment by authorisation** [comment]
> Perhaps I think a switch is needed in snapmod to turn off snapmod and sort of unlink it from the official snap and then turn it back on when needed. Since sending video works on normal snap.

There is, check the LSPosed Manager

**Comment by Lucky-069** [comment]
> > Perhaps I think a switch is needed in snapmod to turn off snapmod and sort of unlink it from the official snap and then turn it back on when needed. Since sending video works on normal snap.
> 
> There is, check the LSPosed Manager

If you checked my comment. I did say we need it natively in the app and not having to go to lsposed manager everytime to turn it on and off!

**Comment by RevealedSoulEven** [comment]
Hmmmm please don't go off-track! The thing is that me myself is using snapchat 11.92.0.33 for now and the mappings are available in my github repositories. You can use it.
But I started this "Issue" just to make the developer aware of the situation and fix it in the upcoming updates.

Thank you!
Btw as I always say❤️Great Project

**Comment by authorisation** [comment]
> > > Perhaps I think a switch is needed in snapmod to turn off snapmod and sort of unlink it from the official snap and then turn it back on when needed. Since sending video works on normal snap.
> > 
> > 
> > There is, check the LSPosed Manager
> 
> If you checked my comment. I did say we need it natively in the app and not having to go to lsposed manager everytime to turn it on and off!

Not necessary.

**Comment by Radzi11** [comment]
> Hmmmm please don't go off-track! The thing is that me myself is using snapchat 11.92.0.33 for now and the mappings are available in my github repositories. You can use it. But I started this "Issue" just to make the developer aware of the situation and fix it in the upcoming updates.
> 
> Thank you! Btw as I always say❤️Great Project

your version works perfectly thx.

**Comment by RevealedSoulEven** [comment]
@Radzi11 your welcome

**Comment by IkerFuenmayor** [comment]
> > No bro. It's a bug which I noticed is occurring due to snapmod from snapchat 11.93.xx.xx update because I used to build the mappings myself so I know it. Everything is working fine if I use snapchat 11.92.0.33 with mappings(self made)
> 
> yeah i have the exactly same bug. In the new Snapchat 11.96. . Privacy policy is changed with some new feautures. So the feauture of sending gallery video as snap isn't working. Snap failed to be sent is always shown on mine too. So i changed back my Snapchat to older version and used old mapping of 84615 which for Snapchat 11.82.0.32. Sending gallery video is working fine on the older build. Here is the mapping of 11.82.0.32 attached below. Unzip and Just copy it to/data/data/xyz.rodit.snapmod/files/. For time being i don't have any issue with older version. It works perfectly fine. [84615.zip](https://github.com/rodit/SnapMod/files/9638416/84615.zip)

But it tells me that version 11.82.0.32 and 11.92.0.33 are not compatible. Do you think it's the app? Where can I download the correct version of Snapchat? (I use the latest version of snapmod)
![IMG_20220925_132730](https://user-images.githubusercontent.com/20551283/192159532-f12ef443-1ec7-4237-942a-c1ca0aa79904.jpg)

**Comment by authorisation** [comment]
> > > No bro. It's a bug which I noticed is occurring due to snapmod from snapchat 11.93.xx.xx update because I used to build the mappings myself so I know it. Everything is working fine if I use snapchat 11.92.0.33 with mappings(self made)
> > 
> > 
> > yeah i have the exactly same bug. In the new Snapchat 11.96. . Privacy policy is changed with some new feautures. So the feauture of sending gallery video as snap isn't working. Snap failed to be sent is always shown on mine too. So i changed back my Snapchat to older version and used old mapping of 84615 which for Snapchat 11.82.0.32. Sending gallery video is working fine on the older build. Here is the mapping of 11.82.0.32 attached below. Unzip and Just copy it to/data/data/xyz.rodit.snapmod/files/. For time being i don't have any issue with older version. It works perfectly fine. [84615.zip](https://github.com/rodit/SnapMod/files/9638416/84615.zip)
> 
> But it tells me that version 11.82.0.32 and 11.92.0.33 are not compatible. Do you think it's the app? Where can I download the correct version of Snapchat? (I use the latest version of snapmod) ![IMG_20220925_132730](https://user-images.githubusercontent.com/20551283/192159532-f12ef443-1ec7-4237-942a-c1ca0aa79904.jpg)

You have to install the mappings for older Snapchat versions manually.

**Comment by IkerFuenmayor** [comment]
Yes, ignore that fact, it has already been corrected. However, when sending photos from the gallery, it continues indicating that they are from the "photos" section, it does not send them as live. What version is the one that supports this?

**Comment by authorisation** [comment]
> Yes, ignore that fact, it has already been corrected. However, when sending photos from the gallery, it continues indicating that they are from the "photos" section, it does not send them as live. What version is the one that supports this?

From what I know it works on 11.82.0.32

**Comment by IkerFuenmayor** [comment]
> > Yes, ignore that fact, it has already been corrected. However, when sending photos from the gallery, it continues indicating that they are from the "photos" section, it does not send them as live. What version is the one that supports this?
> 
> From what I know it works on 11.82.0.32

Yes, within the chats as a message it is sent as live, but if it was published as a normal story, it indicates that it is taken from photos.

**Comment by tycoon420** [comment]
> Hmmmm please don't go off-track! The thing is that me myself is using snapchat 11.92.0.33 for now and the mappings are available in my github repositories. You can use it. But I started this "Issue" just to make the developer aware of the situation and fix it in the upcoming updates.
> 
> Thank you! Btw as I always say❤️Great Project

bro is saving working on ur the version 11.92.0.33 rn? i dont know why saving pics and videos stopped on 11.82.0.32  20 mins ago. :(

**Comment by Sadyantssr** [comment]
> Hmmmm please don't go off-track! The thing is that me myself is using snapchat 11.92.0.33 for now and the mappings are available in my github repositories. You can use it. But I started this "Issue" just to make the developer aware of the situation and fix it in the upcoming updates.
> 
> Thank you! Btw as I always say❤️Great Project

Sir, I can't find any mappings for the version 11.92.0.33 in your repository, as you mentioned earlier

**Comment by RevealedSoulEven** [comment]
> > Hmmmm please don't go off-track! The thing is that me myself is using snapchat 11.92.0.33 for now and the mappings are available in my github repositories. You can use it. But I started this "Issue" just to make the developer aware of the situation and fix it in the upcoming updates.
> > Thank you! Btw as I always say❤️Great Project
> 
> Sir, I can't find any mappings for the version 11.92.0.33 in your repository, as you mentioned earlier

Check my repositories and use the latest snapmod with the mappings provided. It's for 11.92.0.33
